e2e/browser/test-app/package.json

```diff
@@ -9,19 +9,19 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@inrupt/solid-client-vc": "^0.5.0",
-    "next": "13.5.4",
+    "@inrupt/solid-client-vc": "file:../../../",
+    "next": "^14.0.4",
     "react": "18.2.0"
   },
   "devDependencies": {
-    "@inrupt/solid-client-authn-browser": "^1.12.3",
-    "@inrupt/eslint-config-lib": "^1.5.3",
-    "@inrupt/solid-ui-react": "^2.8.2",
-    "@types/node": "18.11.12",
-    "@types/react": "18.0.26",
-    "@types/react-dom": "18.0.9",
-    "eslint": "8.29.0",
-    "eslint-config-next": "13.0.6",
-    "typescript": "4.9.4"
+    "@inrupt/eslint-config-lib": "^3.0.1",
+    "@inrupt/solid-client-authn-browser": "^1.17.5",
```

Comment:
Can use v2 of authn browser

Note `npx npm-check-updates` will show you any packages you can update and `npx npm-check-updates -u` will perform the updates.

Reply:
Though I would only do upgrades of devDependencies in the package.json

Reply:
Done in [385fd3c](https://github.com/inrupt/solid-client-vc-js/pull/899/commits/385fd3c030dd2c980b7ed5fccf19d2dc9538b6a2), and also I realized I didn't complete all of the things depcheck was complaining about [cc1a58e](https://github.com/inrupt/solid-client-vc-js/pull/899/commits/cc1a58efda8b69fc188b469a563baf6d7fa1774f)